includes/wikia/tests/GlobalTitleTest.php

```diff
@@ -17,11 +17,14 @@ function setUp() {
 				[ 'wgServer', 113, 'http://memory-alpha.wikia.com' ],
 				[ 'wgServer', 490, 'http://wowwiki.wikia.com' ],
 				[ 'wgServer', 1686, 'http://spolecznosc.wikia.com' ],
-				[ 'wgExtraNamespacesLocal', 490, [ 116 => 'Portal' ] ],
 				[ 'wgLanguageCode', 177, 'en' ],
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "wgLanguageCode" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)